lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureBlobType.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.IO.FileSystem.AzureBlob
+{
+    /// <summary>
+    /// The type of blob supported for the <see cref="IFileSystem"/>.
+    /// For more details on each type of blob, see https://msdn.microsoft.com/en-us/library/azure/ee691964.aspx.
+    /// </summary>
+    public enum AzureBlobType
```

Comment:
I don't think you can seal an enum. I couldn't find an enum in the Azure library.